### PR TITLE
[Doc] Add documentation for `<DatagridAG>`

### PR DESCRIPTION
**Pull request by slax57**

### CODE REVIEW

File: docs/DatagridAG.md [hunk 1]
```diff
@@ -0,0 +1,762 @@
...
+---
+
+# `<DatagridAG>`
+
+This [Enterprise Edition](https://marmelab.com/ra-enterprise)<img class="icon" src="./img/premium.svg" /> component is an alternative datagrid component with advanced features, based on [ag-grid](https://www.ag-grid.com/).
+
+> **Note:** This component is still in **alpha** stage. Major changes may still be introduced in the future.
+
+<video controls autoplay playsinline muted loop>
+  <source src="https://marmelab.com/ra-enterprise/modules/assets/DatagridAG.mp4" type="video/mp4"/>
```
Comment (adguernier) [inline docs/DatagridAG.md:13]: The video does not work in Firefox

File: docs/DatagridAG.md [hunk 2]
```diff
@@ -0,0 +1,762 @@
...
+-   Tree Data
+-   Pivoting
+-   More advanced filtering
+-   Master Detail views
+-   Range Selection
+-   Excel Export
+-   And more...
+
+<video controls autoplay playsinline muted loop>
+  <source src="https://marmelab.com/ra-enterprise/modules/assets/DatagridAG-enterprise.mp4" type="video/mp4"/>
```
Comment (adguernier) [inline docs/DatagridAG.md:41]: Does not work on Firefox

File: docs/DatagridAG.md [hunk 3]
```diff
@@ -0,0 +1,762 @@
...
+                rowGroupPanelShow="always"
+                groupSelectsChildren
+            />
+        </List>
+    );
+};
+```
+
+<video controls autoplay playsinline muted loop>
+  <source src="https://marmelab.com/ra-enterprise/modules/assets/DatagridAG-enterprise.mp4" type="video/mp4"/>
```
Comment (adguernier) [inline docs/DatagridAG.md:760]: Does not work on Firefox

**Review by adguernier — CHANGES_REQUESTED** [review]

File: docs/DatagridAG.md (hunk 1 above)
Comment (fzaninotto) [reply docs/DatagridAG.md:13]: This is fixed in the ra-e,terprise doc.

**Review by fzaninotto — APPROVED** [review]

**Comment by fzaninotto** [comment]
To let users find DatagridAG when they aren't looking for it, I also added various links to that page from other places in the doc. 

See ddde9f75d18ffbf3de9cacb526eb5850641489fe

**Pull request merged**